### PR TITLE
[Technical Support] LPS-76292 Guard against the case the original author of the structure has been deleted

### PR DESCRIPTION
**Pull request by cleydyr**
Is it okay?

See https://issues.liferay.com/browse/LPS-76292

### CODE REVIEW

**Comment by cleydyr** [comment]
ci:test

**Comment by pedroqueiroz94** [comment]
ci:retest

**Comment by pedroqueiroz94** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/service/impl/DDMStructureLocalServiceImpl.java [hunk 1]
```diff
@@ -1530,8 +1530,8 @@ protected DDMStructureVersion addStructureVersion(
 
 		structureVersion.setGroupId(structure.getGroupId());
 		structureVersion.setCompanyId(structure.getCompanyId());
-		structureVersion.setUserId(structure.getUserId());
-		structureVersion.setUserName(structure.getUserName());
+		structureVersion.setUserId(user.getUserId());
+		structureVersion.setUserName(user.getUserName());
```
Comment (pedroqueiroz94) [inline dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/service/impl/DDMStructureLocalServiceImpl.java]: Hey @cleydyr , we're getting a compilation error here. :( It seems that `getUserName` is not defined in `user`. Can you, please, check that? Thank you!

**Comment by pedroqueiroz94** [comment]
Hey @cleydyr , we're getting a compilation error. :/ We will close this PR, ok? 

Please, check my comment.

**Comment by pedroqueiroz94** [comment]
ci:close

File: dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/service/impl/DDMStructureLocalServiceImpl.java (hunk 1 above)
Comment (cleydyr) [reply dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/service/impl/DDMStructureLocalServiceImpl.java]: That's true. Sorry for that. I did it correctly on the main repo, but typed manually when changing the separate repo.

**Pull request closed without merging**